### ISSUE TITLE
Label context support

### ISSUE THREAD

**Issue by adys** [cloudposse/terraform-aws-dynamodb#67]
## Describe the Feature

Hi, would it make sense to update this module to support label context?

Are you guys planning to update all of your modules to support context using some kind of automation?

If you don't, I can prepare a PR where I introduce the context to this module. See [aws-s3-bucket context PR](https://github.com/cloudposse/terraform-aws-s3-bucket/pull/54/files) as a reference.

**Comment by aknysh** [comment]
@adys 
Thanks, yes we are working on adding `context` to all our modules (and some other new features).
We have many modules, so yes your help would be welcome and appreciated.
PRs are always welcome.
If you want to take a stub at this, please run the following commands in the root of the repo:

```
make init
make github/init
cd test/src/
go mod init github.com/cloudposse/$(cd ../../ && basename `pwd`)
go mod tidy
git rm Gopkg.*
git add go.*
cp ../../../terraform-null-label/test/src/Makefile 
```

We need to add `context.tf` to the module and to the examples/complete.
This also will update the Terratest to use `go` modules (if that was not updated yet).

Thank you

**Comment by adys** [comment]
@aknysh I just noticed this module depends on [aws-dynamodb-autoscaler](https://github.com/cloudposse/terraform-aws-dynamodb-autoscaler) module which doesn't support the context either.

I've created [PR](https://github.com/cloudposse/terraform-aws-dynamodb-autoscaler/pull/25) to add context to the dependency module first. Once it gets merged, I'll update this one as well.

**Comment by adys** [comment]
@aknysh any chance someone can take a look at [PR](https://github.com/cloudposse/terraform-aws-dynamodb-autoscaler/pull/25) so I can finish this one as well? Thank you.

**Comment by Nuru** [comment]
Closed by #72
